### ISSUE TITLE
Wildcard in Ignored Segments

### ISSUE THREAD

**Issue by matthewmrotter** [sjelfull/craft-countryredirect#23]
**Is your feature request related to a problem? Please describe.**

The ignored segments section doesn't allow wildcards in the urls 

```
  'ignoreSegments'      => [
  '/resources/case-studies'
 ],
```

will ignore /resources/case-studies/xxxxx

but will also ignore /resources/case-studies

which is not the desired behavior

**Describe the solution you'd like**

Allow wild cards 

```
  'ignoreSegments'      => [
  '/resources/case-studies/*'
 ],
```

should redirect all urls that begin with the segment /resources/case-studies but not the main page.

**Comment by sjelfull** [comment]
Thanks for reaching out. I'm working on a bigger release that will be out soon, and will take a look at this after that.

In general terms, its using [`strpos()`](https://www.php.net/manual/en/function.strpos.php) under the hood - which doesn't support wildcards.

This is a good idea, so I will plan to add it.

**Comment by sjelfull** [comment]
This was fixed in 2.2.0 with the addition of `ignoreUrlPatterns` which support regex and exact matches.
